### PR TITLE
[sync] 2023/11/15

### LINKED ISSUE #436: [PAGE ISSUE]: 'Dart 开发语言概览'
### 页面 URL

https://dart.cn/guides/language/language-tour

### 页面源地址

https://github.com/cfug/dart.cn/tree/master/src/_guides/language/language-tour.md

### 描述问题

image
并不是必须有 `break` 语句，没有也可以。

image
根据[英文官网](https://dart.dev/language/branches#switch-statements)来看，非空的语句执行完会自动跳出 `switch` 

### 期望如何修复

_No response_

### 附加信息

_No response_

### LINKED ISSUE #440: [PAGE ISSUE]: '创建 package'
### 页面 URL

https://dart.cn/guides/libraries/create-packages

### 页面源地址

https://github.com/cfug/dart.cn/tree/master/src/_guides/libraries/create-packages.md

### 描述问题

词组错误：位于`组织package的代码结构`章节下的这个段落，`最佳新能`词组错误。

段落具体如下：
为了在开发时使 [dartdevc](https://dart.cn/tools/dartdevc) 工具能够达到最佳新能，应该将 [实现文件](https://dart.cn/tools/pub/package-layout#implementation-files) 放到目录 /lib/src 下，而不是 /lib 目录的其他地方。另外，避免通过 package:package_name/src/... 导入文件。

### 期望如何修复

`最佳新能`应当为`最佳性能`。

### 附加信息

_No response_

### PR DESCRIPTION
**Pull request by AmosHuKe**
ref #123 

- fix #440 
- fix #436

### CODE REVIEW

**Comment by AlexV525** [comment]
> Error processing "/home/runner/work/dart.cn/dart.cn/src/sitemap.xml": PathNotFoundException: Cannot open file, path = '/home/runner/work/dart.cn/dart.cn/src/sitemap.xml' (OS Error: No such file or directory, errno = 2)

**Comment by AmosHuKe** [comment]
Verified locally
83c8fefb62335024a761ba46d1e04c0c

**Comment by AmosHuKe** [comment]
cc @AlexV525 @chenglu

**Comment by chenglu** [comment]
👍

**Review by chenglu — APPROVED** [review]
👍 辛苦啦

**Comment by AmosHuKe** [comment]
@chenglu 麻烦路哥啦，部署没法测试，不过看起来没啥问题。

File: _config_dev.yml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 # Site settings for development mode
-# url: https://dartlang-org-dev.firebaseapp.com
-# url: https://dartlang-org-staging-0.firebaseapp.com
 
 # Dev url inside Docker container
 url: http://0.0.0.0:4000
```
Comment (AlexV525) [inline _config_dev.yml:4]: 顺便改成 local 吧

```suggestion
url: http://127.0.0.1:4000
```

**Review by AlexV525 — APPROVED** [review]

**Pull request merged**